doc/demo.rst

```diff
@@ -687,73 +785,87 @@ Auto-importing files and temporary names
+whose first character is a ``.`` (dot). Note: this is only true of *file*
 names: alpenhorn is still willing to import paths which contain
 *directories* with leading dots in their names, assuming such names are
 acceptable to at least one of your import extensions.
 
-As an example, let’s create a ``.dat`` file with a temporary name by
-appending, say, ``.temp`` to the name of the file we want to create:
+As an example, let's create a ``.dat`` file with a temporary name by
+appending, say, ``.temp`` to the name of the file we want to create.
+In the ``alpenhost1`` container:
 
-::
+.. code:: console
+   :class: democontainer
 
-   mkdir /data/2025/02/21/23
-   echo "0 1 2 3 4 5" > /data/2025/02/21/23/1324.dat.temp
+   cd /data
+   mkdir 2025/02/21/23
+   echo "0 1 2 3 4 5" > 2025/02/21/23/1324.dat.temp
 
 This file creation will be noticed by alpenhorn, but no import will
-occur, because the ``pattern_exporter`` won’t accept the name as valid:
+occur, because the ``pattern_exporter`` won't accept the name as valid:
 
-::
+.. code:: console
+   :class: demohost
 
-   alpen1-1  | Feb 21 23:51:59 INFO >> [Worker#1] Beginning task Import 2025/02/21/23/1324.dat.temp on demo_storage1
-   alpen1-1  | Feb 21 23:51:59 INFO >> [Worker#1] Not importing non-acquisition path: 2025/02/21/23/1324.dat.temp
-   alpen1-1  | Feb 21 23:51:59 INFO >> [Worker#1] Finished task: Import 2025/02/21/23/1324.dat.temp on demo_storage1
+   alpenhost1-1  | Feb 21 23:51:59 INFO >> [Worker#1] Beginning task Import 2025/02/21/23/1324.dat.temp on demo_storage1
+   alpenhost1-1  | Feb 21 23:51:59 INFO >> [Worker#1] Not importing non-acquisition path: 2025/02/21/23/1324.dat.temp
+   alpenhost1-1  | Feb 21 23:51:59 INFO >> [Worker#1] Finished task: Import 2025/02/21/23/1324.dat.temp on demo_storage1
 
-The message “Not importing non-acquisition path” means no import
+The message "Not importing non-acquisition path" means no import
 extension indicated to alpenhorn that the file should be imported. If,
 instead, we had used a temporary filename with a leading dot, say,
-``/data/2025/02/21/23/.1324.dat``, an import task wouldn’t have even
+``/data/2025/02/21/23/.1324.dat``, an import task wouldn't have even
 been made, since alpenhorn would have rejected the file name earlier,
 before it got around to attempting to import the file.
 
 After file is fully written, it can be moved to the correct name. On
 most filesystems, this is an atomic operation:
 
-::
+.. code:: console
+   :class: democontainer
+
+   mv 2025/02/21/23/1324.dat.temp 2025/02/21/23/1324.dat
 
-   mv /data/2025/02/21/23/1324.dat.temp /data/2025/02/21/23/1324.dat
+.. hint::
+   By "atomic operation" we mean: on most filesystems there is never
+   a time during execution of the ``mv`` command when the destination
+   filename ``2025/02/21/23/1324.dat`` refers to a partial file.  Either
+   the destination file doesn't exist, or it exists and is complete.
```

Comment:
> -> Why do you mention the point about an atomic operation? In case the file is created before its contents are written in a `mv` command?

Definitely worth expanding on that term. Here's a hint.